types/inet.go

```diff
@@ -154,6 +149,13 @@ func (a *InetArray) Value(i int) *net.IPNet {
 	if a.IsNull(i) {
 		return nil
 	}
+	cidr := string(a.Storage().(*array.Binary).Value(i))
+	if len(cidr) == 0 {
+		return &net.IPNet{
+			IP:   net.IPv4zero,
+			Mask: make(net.IPMask, 4),
```

Comment:
What is the purpose of `4` here or where is it coming from?

Reply:
the `ipv4Len` is private in `net`, but this is just to make the mask of the same size. I can replace this with the len of `net.IPv4zero`

Reply:
Done in a8076df52d738719811bd89c894f3c2a11f1f512